### PR TITLE
added how to run datadog locally using docker

### PR DESCRIPTION
**Pull request by mchan9125**

### CODE REVIEW

File: datadog/datadog_run_local_agent/datadog_run_local_with_docker.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Overview
+
+This snippet describes how to run a local DataDog agent in Docker. Note for when running locally: the
+collection of data has not yet been tested. 
+
+#Prerequisites
+
+1. Have a DataDog [API Key](https://app.datadoghq.com/organization-settings/api-keys?_gl=1*1idzuq7*_gcl_aw*R0NMLjE2NDM5MzY2NzAuQ2p3S0NBaUFsLTZQQmhCQ0Vpd0FjMkdPVkZBeXFIS2J0OEJxUGZRQjRSdV90MXYyNEJlNy1JTFJKaHlEX0cwdzZHNHNoU2RhMXBOcExSb0NBWHdRQXZEX0J3RQ..*_ga*MTcyNzYyMzc5OC4xNjQzODQzNzk3*_ga_KN80RDFSQK*MTY0NDI3MTAyOS4xMS4xLjE2NDQyNzM2OTkuMA..)
```
Comment (colinbjohnson) [inline datadog/datadog_run_local_agent/datadog_run_local_with_docker.md:8]: This URL might not be quite right.

File: datadog/datadog_run_local_agent/datadog_run_local_with_docker.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Running
+
+## Install Datadog Agent in Docker
+
+```
+docker run -d --name dd-agent -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e DD_API_KEY=<DATADOG_API_KEY> gcr.io/datadoghq/agent:7
```
Comment (colinbjohnson) [inline datadog/datadog_run_local_agent/datadog_run_local_with_docker.md:15]: the `-v` volumes aren't really relevant on macOS or to testing - as these exist (I think) primarily on Linux systems. We can also/should also be careful where we get the datadog image from (the one utilized above one comes from Google's DataDog image repository: `gcr.io/datadoghq/agent:7`) -> so we can use one from regular DockerHub (`datadog/agent`).

File: datadog/datadog_run_local_agent/datadog_run_local_with_docker.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+This snippet describes how to run a local DataDog agent in Docker. Note for when running locally: the
+collection of data has not yet been tested. 
+
+#Prerequisites
+
+1. Have a DataDog [API Key](https://app.datadoghq.com/organization-settings/api-keys?_gl=1*1idzuq7*_gcl_aw*R0NMLjE2NDM5MzY2NzAuQ2p3S0NBaUFsLTZQQmhCQ0Vpd0FjMkdPVkZBeXFIS2J0OEJxUGZRQjRSdV90MXYyNEJlNy1JTFJKaHlEX0cwdzZHNHNoU2RhMXBOcExSb0NBWHdRQXZEX0J3RQ..*_ga*MTcyNzYyMzc5OC4xNjQzODQzNzk3*_ga_KN80RDFSQK*MTY0NDI3MTAyOS4xMS4xLjE2NDQyNzM2OTkuMA..)
+
+# Running
+
+## Install Datadog Agent in Docker
```
Comment (colinbjohnson) [inline datadog/datadog_run_local_agent/datadog_run_local_with_docker.md:12]: We can actually remove the `## Install Datadog Agent in Docker` section.

File: datadog/datadog_run_local_agent/datadog_run_local_with_docker.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+docker run --publish 8126:8216 --env DD_API_KEY=<DATADOG_API_KEY> datadog/agent 
+```
+# Confirming
+
+Confirm that the container is successfully running:
+
+```
+docker ps
```
Comment (colinbjohnson) [inline datadog/datadog_run_local_agent/datadog_run_local_with_docker.md:27]: Might be worthwhile describing a bit more what we would look for as an output - for instance, that a container based on the image `datadog:latest` is running.

File: datadog/datadog_run_local_agent/datadog_run_local_with_docker.md (hunk 1 above)
Comment (mchan9125) [reply datadog/datadog_run_local_agent/datadog_run_local_with_docker.md:8]: I see - I stole it from the datadog website. Should I just leave out the url completely?

**Pull request merged**